CriticalMassTests/NextRideManagerTests.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  CriticalMapsTests
+
+import CoreLocation
+@testable import CriticalMaps
+import XCTest
+
+class NextRideManagerTests: XCTestCase {
+    var sut: NextRideManager!
```

Comment:
What does `sut` mean? I would prefer `nextRideManager` as variable name.

Reply:
it usually refers to `system under test` but we have a very loose naming pattern for that

Reply:
Oh, didn’t know that! Now it makes a lot more sense 🙈